test/e2e/predictor/test_canary.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2019 kubeflow.org.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+from kubernetes import client
+
+from kfserving import KFServingClient
+from kfserving import constants
+from kfserving import V1alpha2EndpointSpec
+from kfserving import V1alpha2PredictorSpec
+from kfserving import V1alpha2TensorflowSpec
+from kfserving import V1alpha2InferenceServiceSpec
+from kfserving import V1alpha2InferenceService
+from kubernetes.client import V1ResourceRequirements
+
+from ..common.utils import predict
+from ..common.utils import KFSERVING_TEST_NAMESPACE
+
+api_version = constants.KFSERVING_GROUP + '/' + constants.KFSERVING_VERSION
+KFServing = KFServingClient(config_file="~/.kube/config")
```

Comment:
As I understand it, the KFServingClient should default to this config file. Is this not the case?

Reply:
Yes, it is default to ~/.kube/config. Since SDK is running in a different cluster than KFServing, [user must set config_file to load kube-config file explicitly. ](https://github.com/kubeflow/kfserving/blob/master/python/kfserving/docs/KFServingClient.md)
`Note that for the case that the SDK is running in cluster and you want to operate KFServing in another remote cluster, user must set config_file to load kube-config file explicitly, e.g. KFServingClient(config_file="~/.kube/config").`

Reply:
Without doing this, presubmit test will fail.